jaqpotpy/doa/doa.py

```diff
@@ -185,3 +185,67 @@ def predict(self, new_data: np.array) -> Iterable[Any]:
             self._doa.append(new_data)
             self._in.append(in_doa)
         return doaAll
+
```

Comment:
regarding the 2nd comment, I am not sure about that. Actually, it was something i wanted to ask because it was not somehow explained why we use +- 4*std